ui/.eslintrc.js

```diff
@@ -0,0 +1,88 @@
+module.exports = {
+  root: true,
+  parser: '@typescript-eslint/parser',
+  plugins: [
+    'react',
+    'sort-keys-fix',
+    'prefer-arrow',
+    '@typescript-eslint',
+    'simple-import-sort',
+    'prettier'
+  ],
+  extends: [],
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true 
+    }
+  },
+  settings: {
+    react: {
+      version: 'detect' 
+    }
+  },
+  rules: {
+    // 'prettier/prettier': 'warn',
+
+    /* Typescript specific */
```

Comment:
Was this list of rules sourced from somewhere? Might be worth noting a source if there's some meaningful provenance.

Reply:
I ported the rules over from our common-ui/tsling.json file using the [tslint migration roadmap](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md)

I added a comment to the file noting this - thanks!